src/renderers/shaders/ShaderChunk.js

```diff
@@ -17,34 +18,38 @@ import color_pars_vertex from './ShaderChunk/color_pars_vertex.glsl.js';
 import color_vertex from './ShaderChunk/color_vertex.glsl.js';
 import common from './ShaderChunk/common.glsl.js';
 import cube_uv_reflection_fragment from './ShaderChunk/cube_uv_reflection_fragment.glsl.js';
+import default_fragment from './ShaderChunk/default_fragment.glsl.js';
```

Comment:
I still think we shouldn't add these to `ShaderChunk`.

Reply:
Ok so maybe i didn't understand the usage of default_fragment and default_vert ? Aren't they used by default shader in case nothing is provide ? 

In your view why should they be apart others shaders ?